app/views/courses/_course_table_row.html.erb

```diff
@@ -0,0 +1,37 @@
+<tr class="govuk-table__row">
+  <td class="govuk-table__cell" data-qa="courses-table__course">
+    <%= govuk_link_to "#{ course.attributes[:name] } (#{ course.attributes[:course_code] })",
+      manage_ui_course_page_url(provider_code: @provider[:institution_code], course_code: course.attributes[:course_code]),
+      class: "govuk-link govuk-heading-s govuk-!-margin-bottom-0"
+    %>
+    <span class="govuk-body-s"><%= course.attributes[:description] %></span>
+  </td>
+  <td class="govuk-table__cell" style="border: 3px solid red">
+    Running
+  </td>
+  <td class="govuk-table__cell">
+    <%= render partial: 'content_status_tag' %>
+  </td>
+  <td class="govuk-table__cell" data-qa="courses-table__findable">
+    <% if course.attributes[:findable?] %>
+      <a class="govuk-link" href="#TODO" style="border: 3px solid red">Yes - view online</a>
```

Comment:
Isn’t this a `setting` now that we can pull into the view and pretend the provider code and course code?

Reply:
I will do this on a follow-up PR - keep this one as a refactoring.